packages/react-scripts/scripts/eject.js

```diff
@@ -35,6 +36,30 @@ inquirer
     default: false,
   })
   .then(answer => {
+    // Make sure there are no dirty git status
+    function statusSync() {
+      try {
+        let stdout = execSync(`git status --porcelain`).toString();
```

Comment:
I'm curious, what does this do if git is not installed or if the app is using a VCS other than git?